integrations/influxdb_connector/influxdb-timestream-connector/README.md

```diff
@@ -154,28 +153,121 @@ The REST API Gateway ensures all requests are authenticated with SigV4. Any Infl
 
 ## IAM Permissions
 
-### IAM Deployment Permissions
+The following permissions are the least required permissions for deploying and executing the connector. These permissions assume the stack is named `InfluxDBTimestreamConnector`.
 
-[//]: # (TODO: Update deployment policy with least privilege once REST API Gateway deployment has been tested.)
+### IAM Deployment Permissions
 
 The following is the least privileged IAM permissions for deploying the connector.
```

Comment:
Is there SAM documentation we can reference that describes what permissions it needs?

Can SAM-required permissions be split from connector-specific permissions? As-is, it's not clear whether the SAM tool or the connector needs to change account-wide API Gateway settings.

Reply:
As far as I know there are no pre-defined permissions the SAM CLI requires; it depends on the template.

The connector (the stack) needs permissions to do things like add messages to the dead letter queue or create CloudWatch logs. These are defined in the template and created by the SAM CLI. Therefore, the user needs permission to use the SAM CLI to create these resources, including IAM policies and roles.

When you say account-wide API Gateway settings, are you referring to the `RestApiGatewayLogAccount` resource defined in the template or the documentation of the IAM permission for `logs:DescribeLogGroups` on `"arn:aws:logs:{region}:{account-id}:log-group::log-stream:"`?